pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: json_theme_plus
 description: A library to dynamically generate a ThemeData object from a JSON file or dynamic map object
 homepage: https://github.com/zeshuaro/json_theme_plus
-version: 6.6.6
+version: 6.6.7
```

Comment:
There's no need to update this. It'll be automatically updated when the package gets released.

Reply:
Great! I'll get that changed real quick

Reply:
@zeshuaro changes are in, thanks!